CHANGELOG.md

```diff
@@ -1,19 +1,26 @@
 10.0.0 Release notes (2020-3-26)
 =============================================================
-NOTE 1: Requires MacOS v10.14 (Mojave) to build.
+NOTE 1: This release brings many breaking changes and new features.
+NOTE 2: The old Realm Sync (against Realm Cloud and/or on-prim Realm Object Server) has been removed. The new Realm Sync works together with MongoDB Realm.
+NOTE 3: Deprecated methods have been removed.
 
 ### Breaking changes
 * `Realm.Sync.User` has been replaced by `Realm.User`.
-* `Realm.Sync.Adapter`, `Realm.Sync.addlistener()`, `Realm.Sync.localListenerRealms()`, `Realm.Sync.removeAllListeners()` and `Realm.Sync.removeListener()` have been removed. ([RJS-434](https://jira.mongodb.org/browse/RJS-434))
-* Query-based Sync has been removed. This includes `Realm.Sync.Subscription`, `Realm.Results.subscribe()`, `Realm.subscriptions()`, `Realm.unsubscribe()`, `Realm.privileges()`. These APIs are not initially supported by MongoDB Realm. They will be re-introduced in a future release. `partitionValue` has been added to the `sync` configuration as a replacement. Read more [here](XXX). ([]())
+* `Realm.Sync.Adapter`, `Realm.Sync.addlistener()`, `Realm.Sync.localListenerRealms()`, `Realm.Sync.removeAllListeners()` and `Realm.Sync.removeListener()` have been removed. ([#2732](https://github.com/realm/realm-js/issues/2732))
+* Query-based Sync has been removed. This includes `Realm.Sync.Subscription`, `Realm.Results.subscribe()`, `Realm.subscriptions()`, `Realm.unsubscribe()`, `Realm.privileges()`. These APIs are not initially supported by MongoDB Realm. They will be re-introduced in a future release. `partitionValue` has been added to the `sync` configuration as a replacement. Please read the documentation to get the full picture.
```

Comment:
This is not polished. Maybe something more formal: `Read the documentation here (and a reference where is it)`